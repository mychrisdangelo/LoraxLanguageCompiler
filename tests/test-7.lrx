int main()
{

	int a;
	float b;
	bool c;
	char d;
	a = 1;
	b = 3.14;
	c = true;
	d = 'a';
	print(a, b, c, d);
}


