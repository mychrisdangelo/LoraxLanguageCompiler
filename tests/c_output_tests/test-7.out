#include "lrxlib.h"
int main();

int main()
{
int a_1 = 0;
float b_1 = 0.0;
bool c_1 = false;
char d_1 = '\0';
int __tmp_int_5 = 0;
float __tmp_float_4 = 0.0;
bool __tmp_bool_3 = false;
char __tmp_char_2 = '\0';
int __tmp_int_1 = 0;
int __tmp_int_0 = 0;

__tmp_int_5 = 1;

a_1 = __tmp_int_5;

__tmp_float_4 = 3.14;

b_1 = __tmp_float_4;

__tmp_bool_3 = true;

c_1 = __tmp_bool_3;

__tmp_char_2 = 'a';

d_1 = __tmp_char_2;

fprintf(stdout, "%d", a_1);
fprintf(stdout, "%f", b_1);
lrx_print_bool(c_1);
fprintf(stdout, "%c", d_1);

return __tmp_int_0;

}
